src/modules/DataStore/StatisticStore.ts

```diff
@@ -130,12 +130,16 @@ export default class Statistics implements Saveable {
 
                     // If it's not an int or less than zero, we do not want to set it
                     const id: number = Math.floor(+prop);
-                    if (isNaN(id) || id < 0 || id != +prop) {
-                        if (isNaN(id)) {
+                    if (Number.isNaN(id) || id < 0 || id !== +prop) {
+                        if (Number.isNaN(id)) {
+                            // eslint-disable-next-line no-console
                             console.trace(`[Statistics] [${array}] Invalid property requested:`, prop);
```

Comment:
I don't think this does that same thing as it used to.
Fairly certain we need `isNaN` here not `Number.isNaN`

Reply:
I'll double check all of this, but the `isNaN` global [should be avoided](https://github.com/airbnb/javascript#standard-library--isnan) in most cases. I will probably replace some of the places that were coercing numbers with `+prop` just so it's clear and reliable though

Reply:
I've switched some of the places that we coerced numbers with `+prop` to `Number(prop)`. There shouldn't be any real difference with them (although `Number(...)` is more strict than `parseInt(..., 10)` would be), and it will continue to work with casting booleans to their `0` and `1` values